pkg/probe/bgp_neighbors.go

```diff
@@ -0,0 +1,96 @@
+package probe
+
+import (
+	"log"
+	"strconv"
+    
+
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/bluecmd/fortigate_exporter/pkg/http"
+	"github.com/bluecmd/fortigate_exporter/internal/version"
+)
+
+type BGPNeighbor struct {
+	NeighborIP        string       `json:"neighbor_ip"`
+	LocalIP           string       `json:"local_ip"`
+	RemoteAS          int          `json:"remote_as"`
+	AdminStatus       bool         `json:"admin_status"`
+	State             string       `json:"state"`
+}
+
+type BGPNeighborResponse struct {
+	Results    []BGPNeighbor          `json:"results"`
+	VDOM       string                 `json:"vdom"`
+	Path       string                 `json:"path"`
```

Comment:
Please remove the unused fields `path`, `name`, `status` and `build`

Reply:
Fixed :)